src/jsonld/qaPage.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import Head from 'next/head';
+
+import markup from '../utils/markup';
+
+export interface Person {
+  name: string;
+}
+
+export interface Answer {
+  text: string;
+  dateCreated?: string;
+  upvotedCount?: number;
+  url?: string;
+  author?: Person;
+}
+
+export interface Question {
+  name: string;
+  answerCount: number;
+  acceptedAnswer: Answer;
+  suggestedAnswer: Answer[];
```

Comment:
I believe either or is required 🤔 

![Uploading image.png…]()

Reply:
Yeah, I was thinking on this earlier, I guess I just make them both optional 😁